### ISSUE TITLE
az storage blob upload-batch terminates with error in 2.10 but worked in 2.08

### ISSUE THREAD

**Issue by lamaks** [Azure/azure-cli#14660]
## Describe the bug

**Command Name**
`az storage blob upload-batch`

**Errors:**
```
[Errno 2] No such file or directory: 'batch-pool-startup-script'
Traceback (most recent call last):
python3.8/site-packages/knack/cli.py, ln 215, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 654, in execute
    raise ex
cli/core/commands/__init__.py, ln 718, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 709, in _run_job
    cmd_copy.exception_handler(ex)
...
command_modules/storage/operations/blob.py, ln 444, in upload_block_blob
    return client.create_blob_from_path(**create_blob_args)
storage/v2018_11_09/blob/blockblobservice.py, ln 453, in create_blob_from_path
    count = path.getsize(file_path)
../lib/python3.8/genericpath.py, ln 50, in getsize
    return os.stat(filename).st_size
FileNotFoundError: [Errno 2] No such file or directory: 'batch-pool-startup-script'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az storage blob upload-batch -d {} -s {} --account-name {}`

## Environment Summary
```
macOS-10.15.6-x86_64-i386-64bit
Python 3.8.5
Installer: HOMEBREW

azure-cli 2.10.0

Extensions:
application-insights 0.1.1

```

**Comment by JoepDriesen** [comment]
We are experiencing the same issue since yesterday on our CI/CD pipeline.

**Comment by mszykora** [comment]
With the `azure-cli 2.9.1 *` version it still works.

**Comment by yonzhan** [comment]
add to S174

**Comment by justlev** [comment]
@yonzhan This another CI/CD for us as well, and I believe we are not the only ones.
Any chance this can be hotfixed, prior to the S174 release on August 25th?

**Comment by yonzhan** [comment]
@Juliehzl is looking into this and will update here later.

**Comment by Juliehzl** [comment]
dupe of #14641

**Comment by Juliehzl** [comment]
Hi @lamaks , the fix is done in attached PR. Currently you could try available artifacts here https://dev.azure.com/azure-sdk/public/_build/results?buildId=484329&view=artifacts&type=publishedArtifacts with the fix.
I see you are using homebrew installer.  Currently you could try with `pip install xx.whl` in artifacts page first and we will release the hotfix version soon.

**Comment by justlev** [comment]
> Hi @lamaks , the fix is done in attached PR. Currently you could try available artifacts here https://dev.azure.com/azure-sdk/public/_build/results?buildId=484329&view=artifacts&type=publishedArtifacts with the fix.
> I see you are using homebrew installer. Currently you could try with `pip install xx.whl` in artifacts page first and we will release the hotfix version soon.
> 
> Add @justlev and @mszykora

Thank you very much!
We are using the official CircleCI azure-cli orb from here: https://circleci.com/orbs/registry/orb/circleci/azure-cli. The orb, unfortunately, doesn't allow one to explicitly specify a version of the CLI, so we are trying to override the latest version with 2.9.1 using a separate `apt-get install` call

**Comment by Juliehzl** [comment]
Hi all,
We just released hotfix version 2.10.1 and please see https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest for installation.
Homebrew for MacOS will be available tomorrow after the [PR](https://github.com/Homebrew/homebrew-core/pull/59221) merged.
